src/components/VCalendar/mixins/calendar-with-intervals.ts

```diff
@@ -0,0 +1,143 @@
+
+// Mixins
+import CalendarBase from './calendar-base'
+
+// Util
+import { validateNumber } from '../util/validate'
+import {
+  VTimestamp,
+  VTimestampFormatter,
+  copyTimestamp,
+  updateMinutes,
+  createDayList,
+  createIntervalList,
+  createNativeLocaleFormatter
+} from '../util/timestamp'
+
+/* @vue/component */
+export default CalendarBase.extend({
+  name: 'calendar-with-intervals',
+
+  // depends on: maxDays
```

Comment:
what is this referencing?

Reply:
That's no longer accurate, being removed